BackendTaskPilot/BackendTaskPilot.csproj

```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+    <PropertyGroup>
+        <TargetFramework>net7.0</TargetFramework>
```

Comment:
Consider to update to .NET 8.0, which is LTS (long term support): 
https://dotnet.microsoft.com/en-us/download/dotnet